.github/workflows/ci.yml

```diff
@@ -660,10 +657,34 @@ jobs:
 
       - name: Coveralls Finished
         id: publish-coverage
-        uses: coverallsapp/github-action@a1a1a8a300a7e89df3630639df8fb23de5cc6368
+        uses: coverallsapp/github-action@v2
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           parallel-finished: true
 
       - name: Output Coveralls response
         run: echo ${{ steps.publish-coverage.outputs.coveralls-api-result }}
+
+  slack:
+    name: Slack
```

Comment:
should we use a more meaningful name or add a description.

Reply:
Fixed